comp/trace/config/config.go

```diff
@@ -133,6 +133,10 @@ func (c *cfg) SetHandler() http.Handler {
 			httpError(w, http.StatusMethodNotAllowed, fmt.Errorf("%s method not allowed, only %s", req.Method, http.MethodPost))
 			return
 		}
+		if apiutil.Validate(w, req) != nil {
+			return
+		}
+		log.Info("WE VALIDATED THE REQUEST")
```

Comment:
Is this log intended?

Reply:
oops, it is not. I was using it locally to make absolutely sure my test was working as expected